images/uninstall/uninstall.sh

```diff
@@ -2,4 +2,15 @@
 
 kubectl label nodes -l pithos-role=node pithos-role-
 kubectl delete configmap cassandra-cfg pithos-cfg
-kubectl delete -f /var/lib/gravity/resources/pithos.yaml
+kubectl delete -f /var/lib/gravity/resources/cassandra.yaml
+kubectl delete -f /var/lib/gravity/resources/pithos-rc.yaml
+kubectl delete secret cassandra-ssl pithos-keys
+
+for sname in cluster-ca cluster-default-ssl cluster-kube-system-ssl
```

Comment:
We shouldn't delete these things. They can potentially be created by other applications, so we should leave them alone, and simply use them if they already exist.